internal/resource/statefulset_test.go

```diff
@@ -1031,12 +1035,21 @@ default_pass = {{ .Data.data.password }}
 							Expect(sidecar.Args).To(ContainElement(Equal("https://$(HOSTNAME_DOMAIN):15671")))
 						})
 					})
+
+					When("DefaultUserUpdaterImage is set to non-empty string", func() {
+						BeforeEach(func() {
+							instance.Spec.SecretBackend.Vault.DefaultUserUpdaterImage = "another-image"
+						})
+						It("sidecar is configured with the configured image name", func() {
+							Expect(sidecar.Image).To(Equal("another-image"))
+						})
+					})
```

Comment:
The CRD docs say:
> When set to empty string, it disables the sidecar and passwords are not updated in RabbitMQ.

Can we add a unit test for that?

Reply:
No, that was unintentionally left behind.